workflows/offline_ml_diags/offline_ml_diags/_metrics.py

```diff
@@ -50,8 +50,10 @@ def compute_metrics(
     variables, assumed to include variables in
     list arg predicted as well as area and delp
     """
-    column_integrated_vars = [f"column_integrated_{name}" for name in predicted_vars]
-    vertical_bias_vars = [var for var in predicted_vars if var not in ["Q1", "Q2"]]
+    predicted_3d_vars = [var for var in predicted_vars if "z" in ds[var].dims]
```

Comment:
This pattern of `"if "z" in ds[var].dims` is used in a few places to check if a variable is 3D or not.  Would it make sense to write a function called `is_3d` that takes a DataArray as an argument does this check?  I feel like naming it that way makes the intent a little clearer.